antsibull/data/docsite/role.rst.j2

```diff
@@ -0,0 +1,290 @@
+.. Document meta
+
+:orphan:
+
+{# If we can put together source and github repo, we could make the Edit me of github button work.
+   See meta.get("source") in Ansible's docs/docsite/_themes/sphinx_rtd_theme/breadcrumbs.html
+   for more information
+:source: @{ source }@
+-#}
+
+.. Anchors
+
+.. _ansible_collections.@{plugin_name}@_@{plugin_type}@:
+
+.. Anchors: aliases
+
+{# TODO: This assumes that aliases will be short names.
+   If they're FQCN, then we need to change this
+#}
+
+.. Title
+
+{% if entry_points.main and entry_points.main.short_description -%}
+{%   set title = plugin_name + ' -- ' + entry_points.main.short_description | rst_ify -%}
+{% else -%}
+{%   set title = plugin_name -%}
+{% endif -%}
+
+@{ title }@
+@{ '+' * title|length }@
+
+.. Collection note
+
+{% if collection == 'ansible.builtin' -%}
+.. note::
+    This module is part of ``ansible-base`` and included in all Ansible
```

Comment:
Should we be saying it's part of 'ansible-core' at this point?  Also, since I'm not really following the code, is it a module that triggers this note, or is it some future role included in ansible.builtin?

Reply:
I cut this paragraph together to
```
    This role is part of ``ansible-core`` and included in all Ansible
    installations.
```
This case will probably not happen in the short term anyway (and maybe never in the long term as well); I think ansible-core right now doesn't even have a mechanism by which it can contain roles.